views/index/index.php

```diff
@@ -42,14 +56,18 @@
         </tr>
     </thead>
     <tbody>
-    <?php foreach ($this->get('checkout') as $checkout): ?>
+    <?php
```

Comment:
Doesn't look like a visual improvement to me. I liked the previous variant more.